test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -568,7 +589,8 @@ describe('33acrossBidAdapter:', function () {
 
         Object.assign(element, { width: 600, height: 400 });
 
-        expect(spec.buildRequests(bidRequests)).to.deep.equal([ serverRequest ]);
+        const builtRequests = spec.buildRequests(bidRequests);
```

Comment:
This feedback is purely esthetic & optional ..

maybe we can get the 1st array entry from the beginning.
`const [ builtRequest ] = spec.buildRequests(bidRequests);`

Reply:
Done.